README.md

```diff
@@ -94,8 +94,14 @@ sudo snap alias rt-tests.svsematest svsematest
 
 ### Local Build
```

Comment:
```suggestion
### Build and install from source
```

Reply:
I think that "Build and install from source" gives a wrong idea, since it isn't building the source, but only the snap package.